package.json

```diff
@@ -68,6 +68,7 @@
     "build:w3c-debug": "npm run build:w3c -- --debug",
     "build:w3c-with-jquery": "node ./tools/builder.js --profile=w3c-common",
     "build:w3c": "node ./tools/builder.js --profile=w3c",
+    "build:w3c-common": "node ./tools/builder.js --profile=w3c-common",
```

Comment:
We already have `build:w3c-with-jquery` 😁

Reply:
Ahhhh :facepalm: